app/assets/stylesheets/application.tailwind.css

```diff
@@ -34,6 +34,10 @@
   @apply p-0 mt-1 bg-transparent border-none opacity-50 focus:outline-none focus:ring-0 focus-within:opacity-100;
 }
 
+.text-link {
```

Comment:
Generally we're trying to avoid/move away from these types of "roll-up" classes. At least at this stage where things are moving quickly on the styling side.

Reply:
For the record, I'm 100% on board with avoiding traditional CSS "roll-up" classes when using Tailwind. That being said, this is one of those select few I always have since text links are pretty common and it's not worth having a component or something for them. If you're very much against it, I can remove it.

Reply:
I'd prefer to remove for now. We just don't enough established UI convention in place yet to warrant a specific class.